ecosystem/typescript/sdk/src/aptos_account.ts

```diff
@@ -30,6 +32,39 @@ export class AptosAccount {
     return new AptosAccount(HexString.ensure(obj.privateKeyHex).toUint8Array(), obj.address);
   }
 
+  static isValidPath = (path: string): boolean => {
+    if (
+      // eslint-disable-next-line prefer-regex-literals
+      !new RegExp('^m\\/44+\'\\/637+\'\\/[0-9]+\'\\/[0-9]+\'\\/[0-9]+\'+$').test(
+        path,
+      )
+    ) {
+      return false;
+    }
+    return !path
+      .split('/')
+      .slice(1)
+      .map((val: string): string => val.replace('\'', ''))
+      // eslint-disable-next-line no-restricted-globals
+      .some(isNaN as any);
```

Comment:
Why is this needed, if the regex worked?

Reply:
yes. I'll remove it.